src/database.rs

```diff
@@ -228,11 +228,14 @@ fn get_issue_list(range_list: Vec<(IVec, IVec)>) -> Result<Vec<Issue>> {
 
     for (key, val) in range_list {
         let (owner, repo, number) = parse_key(&key)?;
+        let (title, author) = bincode::deserialize::<(String, String)>(&val)
+            .unwrap_or(("No title".to_string(), "No author".to_string()));
```

Comment:
For some reason, there are duplicates of only issue number but no title or author. Is there a way to fix this?
<img width="1493" alt="Screen Shot 2022-08-31 at 4 10 31 PM" src="https://user-images.githubusercontent.com/102562226/187801083-864943ad-6d8d-4623-8610-89aefe913c3b.png">

Reply:
I have no idea where those duplicates came from. Why don't you add a new issue about this problem, including steps to reproduce it? Then we can revisit this later.